package.json

```diff
@@ -61,7 +127,7 @@
     "copy": "shx cp -r dist/src/* dist/esm && shx mv dist/src/* dist && shx rm -rf dist/{src,tests} && downlevel-dts dist dist/ts3.4 && shx cp package.json readme.md LICENSE dist && json -I -f dist/package.json -e \"this.private=false; this.devDependencies=undefined; this.optionalDependencies=undefined; this.scripts=undefined; this.prettier=undefined; this.jest=undefined; this['lint-staged']=undefined;\""
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=12.7.0"
```

Comment:
Shouldn't it be v12.13.0?

Reply:
subpath pattern requires v12.13.0, whereas just subpath requires v12.7.0 according to the docs.
As this is confimred to work in v12.8.0, it should be correct.